specification/agfood/data-plane/Microsoft.AgFoodPlatform/preview/2021-03-31-preview/examples/OAuthTokens_Delete.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
Was this API path removed? Or was this sample never a real API?

Reply:
I think this got replaced with `OAuthTokens_CreateCascadeDeleteJob.json`